### ISSUE TITLE
Cassandra in [cadence-dev] image doesn't start with Docker RAM = 10 GiB

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#1075]
The container logs show the entrypoint script spinning, waiting for Cassandra to start, but this never happens.

The Cadence image works fine with Docker RAM at 2 GiB.  I suspect that the Cassandra startup script is using `free` rather than the cgroup memory limit, although it does appear to be checking whether it's running in a container:
```
    case "`uname`" in
        Linux)
            if `grep -q 'docker\|lxc' /proc/1/cgroup`
            then
                # running in Docker.
                system_memory_in_mb=$((`cat /sys/fs/cgroup/memory/memory.limit_in_bytes`/1024/1024))
                system_cpu_cores=`nproc`
            else
                system_memory_in_mb=`free -m | awk '/:/ {print $2;exit}'`
                system_cpu_cores=`egrep -c 'processor([[:space:]]+):.*' /proc/cpuinfo`
            fi
        ;;
        FreeBSD)
            system_memory_in_bytes=`sysctl hw.physmem | awk '{print $2}'`
            system_memory_in_mb=`expr $system_memory_in_bytes / 1024 / 1024`
            system_cpu_cores=`sysctl hw.ncpu | awk '{print $2}'`
        ;;
        SunOS)
            system_memory_in_mb=`prtconf | awk '/Memory size:/ {print $3}'`
            system_cpu_cores=`psrinfo | wc -l`
        ;;
        Darwin)
            system_memory_in_bytes=`sysctl hw.memsize | awk '{print $2}'`
            system_memory_in_mb=`expr $system_memory_in_bytes / 1024 / 1024`
            system_cpu_cores=`sysctl hw.ncpu | awk '{print $2}'`
        ;;
        *)
            # assume reasonable defaults for e.g. a modern desktop or
            # cheap server
            system_memory_in_mb="2048"
            system_cpu_cores="2"
        ;;
    esac
```

**Comment by jefflill** [comment]
So the problem is with this line:
```
system_memory_in_mb=$((`cat /sys/fs/cgroup/memory/memory.limit_in_bytes`/1024/1024))
```
here's the fix:
```
system_memory_in_bytes=`cat /sys/fs/cgroup/memory/memory.limit_in_bytes`
let system_memory_in_mb=$system_memory_in_bytes/1024/1024
```

**Comment by jefflill** [comment]
FIXED for neonLIBRARY v2.7.0 and the image up on DockerHub was 6 months old, so I went ahead and republished it as well.
